kong/plugins/jwt/handler.lua

```diff
@@ -28,7 +28,12 @@ local function retrieve_token(request, conf)
     end
   end
 
-  local authorization_header = request.get_headers()["authorization"]
+  local authorization_cookie
```

Comment:
This code could be better written as:

```
  local authorization_header = ngx.unescape_uri(ngx.var["cookie_authorization"])
  if not authorization_header then
     authorization_header = request.get_headers()["authorization"]
  end

  if authorization_header then
  ...
```
This will make sure that the cookie authorization has the same flow as the header Authorization